### PR TITLE
ci: add installation of DNN Python Modules

### PR DESCRIPTION
**Pull request by jorendumoulin**
Sorry for breaking the ci with my previous pr #78, this PR fixes it by installing the torch modules before running the tests!

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -90,6 +96,9 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: 'recursive'
+      - name: Install DNN Python Modules
+        run: |
+          pip3 install -r sw/dnn/requirements.txt
```
Comment (JosseVanDelm) [inline .github/workflows/ci.yml:101]: `sw/dnn.yaml` tests are not run here, are you sure this is required?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -138,6 +150,9 @@ jobs:
...
```
Comment (JosseVanDelm) [inline .github/workflows/ci.yml:155]: Same comment here

**Review by JosseVanDelm — APPROVED** [review]
Maybe this is not the most important comment, yet here it is for completeness

**Review by xiaoling-yi — APPROVED** [review]
OK!

**Comment by rgantonio** [comment]
Hallo! I didn't notice early but were these taken out of the `requirements.txt` only because it takes too much space?

**Comment by jorendumoulin** [comment]
Yes! Our github actions were failing because the runners ran out of memory. Taking the unnecessary dnn-requirements out of the container reduces the container size from 12GB to 3GB

**Pull request merged**